.travis.yml

```diff
@@ -1,7 +1,5 @@
 language: php
 
-sudo: false
```

Comment:
the default value for ``sudo`` is not always false actually. It depends on repositories.